refs/biblio.json

```
@@ -5326,6 +5326,12 @@
         "publisher": "W3C WoT Community Group",
         "repository": "https://github.com/webofthings/web-thing-model"
     },
+    "webdriver-bidi": {
+        "href": "https://w3c.github.io/webdriver-bidi/",
+        "publisher": "W3C",
+        "title": "WebDriver BiDi",
+        "status": "Editor’s Draft"
```

Curly quotes are nice but tools that parse the status string could choke on them:

```suggestion
        "status": "Editor's Draft"
```